components/org.wso2.carbon.identity.api.server.webhook.management/org.wso2.carbon.identity.api.server.webhook.management.v1/src/main/resources/templates/enumClass.mustache

```diff
@@ -0,0 +1,34 @@
+@XmlType(name="{{datatypeWithEnum}}")
```

Comment:
Based on the status types will add a enum for status property

Reply:
why this enum class is needed ?

Reply:
for status